lib/ae_mdw/dex.ex

```diff
@@ -211,6 +252,23 @@ defmodule AeMdw.Dex do
     end
   end
 
+  defp build_debug_contract_swaps_streamer(state, create_txi_idx, cursor) do
+    fn direction ->
+      cursor =
+        case cursor do
```

Comment:
Isn't this redundant? Unless I am mistaken, the `create_txi_idx` should also be the same, so basically the cursor always returns itself.

Reply:
That is true, but if we want the cursor to ONLY contain the `{txi, idx}` part then we would need to change `serialize_cursor` as well, so that it doesn't serialize what comes directly from the DB

Reply:
I meant the `case` construct seems redundant, because it returns what it matches.